operations/host_setup.go

```diff
@@ -44,7 +44,7 @@ func hostSetup() cli.Command {
 func runSetupScript(ctx context.Context, wd string, setupAsSudo bool) error {
 	grip.Warning(os.MkdirAll(wd, 0777))
 
-	if _, err := os.Stat(evergreen.SetupScriptName); os.IsNotExist(err) {
+	if err := os.Rename(evergreen.TempSetupScriptName, evergreen.SetupScriptName); os.IsNotExist(err) {
```

Comment:
Things like https://github.com/golang/go/issues/19647 make me think that we should `os.Stat` and check `os.IsNotExist` before we `os.Rename`.

Reply:
I added a stat before this rename.